serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -1 +1,368 @@
+
+| <<retries>>
+| emoji:construction[]
+| {spec_doc_url}#Retry-Definition[Retry Definition]
+
+| <<timeouts>>
+| emoji:construction[]
+| {spec_doc_url}#workflow-timeouts[Workflow Timeouts]
+
+| <<compensation>>
+| emoji:full_moon[]
+| {spec_doc_url}#Workflow-Compensation[Workflow Compensation]
+
+| <<constants>>
+| emoji:full_moon[]
+| {spec_doc_url}#workflow-constants[Workflow Constants]
+
+| <<secrets>>
+| emoji:full_moon[]
+| {spec_doc_url}#workflow-secrets[Workflow Secrets]
+|===
+
+The following sections describe the current status of the supported features.
+
+[#states]
+=== Workflow Model - States
+
+Kogito supports every state described in the specification {spec_version} version.
+
+.Workflow States implementation status detail
+[cols="1,1,1"]
+|===
+| State | Status | Spec Reference
+
+| Event
+| emoji:full_moon[] 
+| {spec_doc_url}#Event-State[Event State]
+
+| Operation
+| emoji:full_moon[] 
+| {spec_doc_url}#Operation-State[Operation State]
+
+| Switch
+| emoji:full_moon[] 
+| {spec_doc_url}#Switch-State[Switch State]
+
+| Delay
+| emoji:full_moon[] 
+| {spec_doc_url}#Delay-State[Delay State]
+
+| Parallel
+| emoji:full_moon[] 
+| {spec_doc_url}#Parallel-State[Parallel State]
+
+| Inject
+| emoji:full_moon[] 
+| {spec_doc_url}#Inject-State[Inject State]
+
+| ForEach
+| emoji:full_moon[]  
+| {spec_doc_url}#ForEach-State[ForEach State]
+
+| Callback
+| emoji:full_moon[] 
+| {spec_doc_url}#Callback-State[Callback State]
+|===
+
+[#functions]
+=== Workflow Model - Functions
+
+The table below illustrates the current status of the functions Kogito supports. 
+
+For additional functions, the specification supports the `custom` type. Therefore, Kogito added support for `sysout` and `java` tyes. See <<func_sysout>> and <<func_java>> sections for more information.
+
+.Workflow Functions implementation status detail
+[cols="1,1,1"]
+|===
+| Function | Status | Spec Reference
+
+| rest
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-restful-service-invocations[Using Functions for RESTful Service Invocations]
+
+| rpc
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-rpc-service-invocations[Using Functions for RPC Service Invocations]
+
+| expression
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-expression-evaluation[Using Functions for Expression Evaluation]
+
+| asyncapi
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-async-api-service-invocations[Using Functions for Async API Service Invocations]
+
+| graphql
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-graphql-service-invocations[Using Functions for GraphQL Service Invocations]
+
+| odata
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-odata-service-invocations[Using Functions for OData Service Invocations]
+
+| custom
+| emoji:full_moon[]
+| {spec_doc_url}#defining-custom-function-types[Defining custom function types]. Kogito supports `java` and `sysout` custom functions.
+|===
+
+[#func_sysout]
```

Comment:
This is a guide about the spec. In my opinion the details of implementing custom functions do not belong here.

Reply:
I thought about that, but since it's too short, I was afraid to break it into a separate guide. I can do that, I was just waiting for more opinions. :)

Please guys, +1 if we break this into another guide or -1 if not.